CardDictionary.php

```diff
@@ -686,6 +686,15 @@ function GoesWhereAfterResolving($cardID, $from = null, $player = "", $playedFro
       return "GY";
     case "DTD202": return "BANISH";
     case "EVO146": return "-";
+    case "HVY143": case "HVY144": case "HVY145":

```

Comment:
I think ideally these should be coded like NAAs where they go into the GY at the framework level rather than having to hardcode each new exception. The same change should fix the card discarded as well.

Reply:
But it's a cost. So it needs to go to the graveyard as a cost. Otherwise, it does not go in line to how they work and will create some problem down the line due to more rules deviation. This way it's the proper way as per the CR and how it works in Flesh and Blood. 

Otherwise, playing this and any instant card that target something in the graveyard will break if they are not there when played. And we get stuck with stuff like attack reactions going on the combat chain months later because we didn't properly code cost/GY it's my feeling. I prefer things to be coded the proper way.